reviewrot/gerritstack.py

```diff
@@ -90,34 +90,69 @@ def check_host_url(self, ssl_verify):
             if response.status_code == 200:
                 return True
             else:
-                raise ValueError('Host URL is incorrectly configured'
-                                 ' in config file.')
+                raise ValueError()
         except (ValueError, requests.ConnectionError):
             raise ValueError('Host URL is incorrectly configured'
                              ' in config file.')
-        except:
-            raise
 
-    def get_comments_count(self, change_id):
+    def get_comments_count(self, comments_response):
         """
-        Check if host url is valid
+        Returns number of comments
+
         Args:
-            change_id (int): pull/merge request id
+            comments_response (dict): dictionary containing
+           comments data of pull request
+
         Returns:
              total_comments(int): Returns count of comments for the
                                   change id
         """
-        request_url = "{}/changes/{}/comments".format(self.url, str(change_id))
-        decoded_response = self._call_api(request_url, ignore_err=True)
         total_comments = 0
-        for response in decoded_response:
-            if response is not '/COMMIT_MSG':
-                messages = decoded_response.get(response)
+        for response in comments_response:
+            if response != '/COMMIT_MSG':
+                messages = comments_response.get(response)
                 total_comments = total_comments + len(messages)
 
         return total_comments
 
-    def format_response(self, decoded_responses, state_, value, duration):
+    def get_last_comment(self, comments_response):
+        """
+        Returns information about last comment of given
+        pull request
+
+        Args:
+           comments_response (dict): dictionary containing
+           comments data of gerrit review
+
+        Returns:
+           last comment (LastComment): Returns namedtuple LastComment
+           with data related to last comment
+        """
+
+        comments = []
+        for response in comments_response:
+            if response != '/COMMIT_MSG':
+                messages = comments_response.get(response)
+                last_comment_date = datetime.strptime(
+                    messages[-1]['updated'][:-3],
```

Comment:
That would deserve an example of a data in a comment.

Reply:
example of whole response, or just decoded messages?

Reply:
Now when you split the expression to multiple lines it makes more sense, but I would rather explain in a comment why there is 'magical' constant [:-3]

Reply:
added comment